scripts/install_third_party_libs.py

```diff
@@ -172,17 +166,16 @@ def compile_protobuf_files(proto_files_paths):
     proto_env['PATH'] += '%s%s/bin' % (os.pathsep, PROTOC_GEN_TS_PATH)
     buf_path = os.path.join(
         BUF_DIR,
-        BUF_DARWIN_FILES[0] if common.is_mac_os() else BUF_LINUX_FILES[0])
-    for path in proto_files_paths:
-        command = [
-            buf_path, 'generate', path]
+        BUF_DARWIN_FILES[0] if common.is_mac_os() else BUF_LINUX_FILES[0])  # type: ignore[no-untyped-call]
+    for path in filepaths:
+        command = [buf_path, 'generate', path]
         process = subprocess.Popen(
-            command, stdout=subprocess.PIPE, stderr=subprocess.PIPE,
+            command,
+            stdout=subprocess.PIPE,
+            stderr=subprocess.PIPE,
             env=proto_env)
-        stdout, stderr = process.communicate()
-        if process.returncode == 0:
```

Comment:
Just as a quick note for why I removed this: on a successful install, it just prints b''. There is no reason to print stdout on successful install if it is empty, it seems to me.

Reply:
Makes sense